test/integration/targets/yum/tasks/main.yml

```diff
@@ -29,49 +15,41 @@
           - sos
         state: absent
 
-    # On CentOS 6 'yum groupremove "Development Tools"' fails when groupremove_leaf_only
-    # isn't enabled, that's why a package belonging to "Development Tools" (autoconf)
-    # is removed instead of the whole group.
     - name: remove installed group
       yum:
-        name: "@Development tools"
+        name: "@Custom Group"
         state: absent
-      when:
-        - (ansible_distribution in ['RedHat', 'CentOS', 'ScientificLinux'] and ansible_distribution_major_version|int > 6) or ansible_distribution in ['Fedora']
-    - name: remove a package from a group
+
+    - name: On Fedora 28 the above results in "Nothing to do" for some reason...
       yum:
-        name: 'autoconf'
+        name: foo
         state: absent
-      when:
-        - ansible_distribution in ['RedHat', 'CentOS', 'ScientificLinux'] and ansible_distribution_major_version|int <= 6
+
   when:
     - ansible_distribution in ['RedHat', 'CentOS', 'ScientificLinux', 'Fedora']
 
-# DNF1 doesn't handle downgrade operations properly (Fedora < 26)
+
 - block:
-    - include: 'repo.yml'
+    - import_tasks: repo.yml
+    - import_tasks: yum_group_remove.yml
+      when: ansible_distribution_major_version|int != 28
```

Comment:
Apparently, on Fedora 28 when removing our custom group the result is `Nothing to do`. We just can't have nice things...

Reply:
Er, nvm.